tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -192,6 +194,46 @@ def ldap_auth_name():
     yield ldap_name
 
 
+def run_command(cmd, hostname=settings.server.hostname, timeout=None):
```

Comment:
we should add it in common helpers. I thought it is already there.

Reply:
We have execute command in helpers, I have written this to have one extra layer checking for return code to have smooth validation. I don't think it can be used multiple files if you think we should add it anyways can be used I will add.

Reply:
I think having it here is good call. Common `run_command` can't throw exception on non-zero return code, as sometimes you want return to be non-zero (e.g. you verify that string is not present in file).

Possibly helpers could have this wrapper, called something like `run_command_assert_success`, but it would require much better name. And I think most of the tests run only one command or two, so benefit of such wrapper is not big for them.

Reply:
@ntkathole do you have more concerns on this? I have added the Test Result